content/en/platform/corda/4.9/community/blob-inspector.md

```diff
@@ -18,13 +18,13 @@ title: Blob Inspector
 
 # Blob inspector
 
-The Corda blob inspector tool gives you a human-readable view of content stored in a [custom binary serialization format](../../../../../en/platform/corda/4.8/open-source/serialization.md).
+The Corda blob inspector tool gives you a human-readable view of content stored in a [custom binary serialization format](serialization.md).
```

Comment:
Should be .html?!

Reply:
I think this is up in the air a bit.. I think @colin-hillis suspects that a new platform would expect files to use the file extension before build so we weren't to waste time switching existing `.md` paths..

Reply:
Yup based on what Colin said I'll leave existing .md links as .md for now.